README.rst

```diff
@@ -1,6 +1,6 @@
-=================================================================
-sqlitedict -- persistent ``dict``, backed-up by SQLite and pickle
-=================================================================
+======================================================================
+sqlitedict -- persistent ``dict``, backed-up by SQLite and pickle/json
```

Comment:
Either keep just `pickle` (default serialization) or remove both `pickle/json` (we support any serialization, implementation detail not fit for package heading). I'd prefer the former:

```suggestion
sqlitedict -- persistent ``dict``, backed up by SQLite
```